.github/workflows/file.yml

```diff
@@ -9,6 +9,8 @@ on:
       - 'logging/**'
       - 'flutter/**'
       - 'dio/**'
+      - 'file/**'
```

Comment:
This is a workflow for the `file` package, I guess here `file/**` should not be ignored?

Reply:
good catch